frontend/src/navigation/NavigationBar.js

```diff
@@ -8,8 +10,6 @@ import HomeRounded from '@material-ui/icons/HomeRounded';
 import PersonRoundedIcon from '@material-ui/icons/PersonRounded';
 import MenuBookRoundedIcon from '@material-ui/icons/MenuBookRounded';
 import LocalFloristRoundedIcon from '@material-ui/icons/LocalFloristRounded';
-import { useState } from 'react';
-import { useHistory } from 'react-router-dom';
```

Comment:
please consider not unnecessarily moving import statements as they blow up your PR and make it more difficult to read